src/components/MobileSortAndFilter/MobileFilterMenu.js

```diff
@@ -0,0 +1,44 @@
+import { useProduct } from "../../contexts";
+
+export default function MobileFilterMenu({ setMobileFilterMenuOpen }) {
+  const {
+    state: { showOnlyInStock, showOnlyFastDelivery },
+    dispatch,
+  } = useProduct();
+  return (
+    <div className="sortfilter-container">
+      <button
+        onClick={() => setMobileFilterMenuOpen(false)}
+        className="close-button btn"
+      >
+        <svg width="1.25rem" height="1.25rem" viewBox="0 0 24 24">
+          <path
+            d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41z"
+            fill="currentColor"
+          ></path>
+        </svg>
+      </button>
+
+      <div className="sortfilter-wrapper">
+        <div className="nav-header">Filter</div>
+        <label>
+          <input
+            checked={showOnlyInStock}
+            type="checkbox"
+            onChange={() => dispatch({ TYPE: "SHOW_ONLY_IN_STOCK" })}
+          />
+          Exclude out of stock
+        </label>
+        <br />
+        <label>
+          <input
+            checked={showOnlyFastDelivery}
+            type="checkbox"
+            onChange={() => dispatch({ TYPE: "SHOW_ONLY_FAST_DELIVERY" })}
+          />
+          Show fast delivery only
+        </label>
+      </div>
+    </div>
+  );
+}
```

Comment:
This is something I learnt and need to implement in my mobile too. I had skipped the filters in mobile view.

Reply:
That's amazing. Happy that it helped you 🙂